app/views/layouts/_page_header.html.erb

```diff
@@ -11,9 +11,12 @@
         userSignedIn: true
       }, 'page_header', JSON.parse('<%= raw escape_json(request_context)%>'))
 
+
       <% if ENV["SEGMENT_JS_ID"] %>
         <!-- Associate user ID with Segment -->
-        analytics.identify(<%= current_user.id %>);
+        analytics.identify(
+          <%= current_user.id %>,
+          <%= raw escape_json(current_user.traits_for_segment) %>);
```

Comment:
out of curiosity do you see this triggering another db request for every request in your local logs?

Reply:
Yes, that's why I did https://github.com/chanzuckerberg/idseq-web/pull/2088 .